src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -1,5 +1,6 @@
 import backendServices from 'platform/user/profile/constants/backendServices';
 import { mhvUrl } from 'platform/site-wide/mhv/utilities';
+import { rootUrl as hearingAidSuppliesFormUrl } from 'applications/disability-benefits/2346/manifest.json';
```

Comment:
I know that we don't seem to import `rootUrl`s in this file but this form's URL will change and I just don't know if we'll remember to update it here